model/src/main/resources/reference.yml

```diff
@@ -0,0 +1,234 @@
+# A reference configuration file containing all possible ORT configuration options. Some of those options are mutually
```

Comment:
The addition of the YAML file raises the question how you converted the file, and how our users are supposed to convert their files. Have you considered to add a `migrate` subcommand to the `helper-cli` that can do the conversion (and possibly assist with other migration tasks in the future)?

Reply:
Ok, in the last commit you mention rightfully that we already have `ort config --show-active` which can be used for the conversion. Maybe already mention `ort config --show-reference` in this commit's message, as I assume that's what you've used for the conversion?

Reply:
I did the conversion by hand to keep the comments and empty lines. It's basically just replacing ` {` with `:` and ` = ` with `: `, and removing `}`.

Reply:
Maybe still worth mentioning this in the commit message?

Reply:
I have added a comment to the commit message mentioning the config command.